WordPress/gutenberg
# Media & Text: Add spacing block supports

Related:

- https://github.com/WordPress/gutenberg/issues/43241
- https://github.com/WordPress/gutenberg/issues/43243

## What?
Add padding and margin support to the Media & Text block.

## Why?
To create consistency across blocks.

## How?
Added the relevant block supports in block.json

## Testing Instructions
1. Insert a new Media & Text block. 
2. Confirm that the dimensions control panel is there and there are no defaults.
3. Add padding and margin.

## Screenshots or screencast 
![media-text-spacing](https://user-images.githubusercontent.com/4832319/185805346-4ec2fe87-f6b2-4d66-9176-c67c492f4abc.gif)
